### ISSUE TITLE
reStructuredText citation references not recognised, citations deleted

### ISSUE THREAD

**Issue by rptb1** [jgm/pandoc#853]
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#citations says

```
Here is a citation reference: [CIT2002]_.

.. [CIT2002] This is the citation.  It's just like a footnote,
   except the label is textual.
```

But pandoc 1.11.1 coverts this to just

```
<p>Here is a citation reference: [CIT2002]_.</p>
```

**Comment by ZelphirKaltstahl** [comment]
Does Pandoc's own citation style work in `rst` files when converting from `rst` to `pdf` in `natbib` `alphadin` style?

**Comment by jgm** [comment]
Pandoc's citation format is only for pandoc markdown. So,
no.

+++ Zelphir Kaltstahl [Nov 22 16 14:18 ]:

**Comment by ZelphirKaltstahl** [comment]
@jgm OK thanks for confirming!

**Comment by jgm** [comment]
We should be able to handle these better than we do now.  We can't treat them as footnotes, for reasons given above, but we could tack on a references section to the end of the document containing them, and generate links to the corresponding references.  So they'd appear like endnotes. This wouldn't be too hard.

**Comment by ZelphirKaltstahl** [comment]
@jgm I simply had to have this citations functionality, so I wrote some conversion scripts in Python. I did not get into writing Pandoc filters (maybe a shame that I did not, but maybe someone can translateit to a Pandoc filter?), but if you think the results could be useful for implementing things in Pandoc (even if only for the regexes or whatever little details there are), you could take a look at the code.
The code is in my repos, search for `rst` and you will find a repo for citations and a repo for internal links of rst.

**Comment by capital-G** [comment]
Sorry for the bump but is it possible to use the references from a bib file within rst like in md? rst has no native concept of citation, the here discussed concept is more a hack by using footnotes, compare https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes and the citation section below.

I currently try to write an academic paper with pandoc (rst+bib=>pdf/html) as I prefer rst over md on more complex documents but it seems that most [cool] features of pandoc are only implemented for markdown (e.g. raw attribute and citation management)?

**Comment by ZelphirKaltstahl** [comment]
@capital-G In case you did not find it, here is my parser for citations: <https://github.com/ZelphirKaltstahl/rst-citations-to-raw-latex> I wrote this years ago and today I would probably think "What horrible code!", so beware. Also I am not sure anyone ever translated this into a proper Pandoc filter.

**Comment by capital-G** [comment]
Thanks for the reply @ZelphirKaltstahl 

> @capital-G In case you did not find it, here is my parser for citations: https://github.com/ZelphirKaltstahl/rst-citations-to-raw-latex

Thanks, I had a look at it already but as far as I understood this can be only used to generate latex output but my goal is to have a unified document through which i can generate a website and a pdf document. Maybe I could use the latex output to generate a website but then why not use use latex directly in the first place, which would save me some trouble of translating rst to latex and only have to worry about latex to html.

I also tried out converting the rst to md, but ran into problems using the markdown dialect, e.g. using raw attributes in combination with a code segment (is this even possible within the md standard? even github md editor fails here)

![image](https://user-images.githubusercontent.com/8267062/224480303-e45b3075-2f1e-409b-9e82-a5ef320c4ab3.png)

> If you are already familiar with Emacs and org-mode: Org-mode has now a citation syntax. I think it makes for a good format for writing academic papers.

Thanks for the hint - I looked into the syntax but I did not find it too compelling. I think if I can't use rst with pandoc I will simply write the paper and latex and manually convert it to a website as it seems the least work :/
